src/App.tsx

```diff
@@ -15,48 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum Classes {
```

Comment:
The `sortBy` name will be more appropriate here 
```suggestion
export enum sortBy {
```